assets/app/index.html

```diff
@@ -112,6 +112,9 @@
     <script src="bower_components/kubernetes-label-selector/labelFilter.js"></script>
     <script src="bower_components/kubernetes-object-describer/dist/object-describer.js"></script>
     <script src="bower_components/openshift-object-describer/dist/object-describer.js"></script>
+    <script src="bower_components/hawtio-utilities/dist/sugar.js"></script>
+    <script src="bower_components/hawtio-utilities/dist/angular-file-upload.js"></script>
```

Comment:
if at all possible, I'd like to avoid adding dependencies for this link extension... I'd rather inline the `findParentWith` function for now